### ISSUE TITLE
Leftovers after deleting Runtime Agent

### ISSUE THREAD

**Issue by janmedrek** [kyma-project/kyma#6958]
**Description**

After deleting Runtime Agent there is a CRD left over that is not affected by Helm Release deletion.

**Expected result**

All resources that are part of / created by Runtime Agent are deleted along with the Helm release.

**Actual result**

Helm returns an error:
```
2020/01/22 21:04:26 Step error:  Details: Helm install error: rpc error: code = Unknown desc = customresourcedefinitions.apiextensions.k8s.io "compassconnections.compass.kyma-project.io" already exists
2020/01/22 21:04:26 install/update error:  Details: Helm install error: rpc error: code = Unknown desc = customresourcedefinitions.apiextensions.k8s.io "compassconnections.compass.kyma-project.io" already exists
2020/01/22 21:04:26 Error during install/update:  Details: Helm install error: rpc error: code = Unknown desc = customresourcedefinitions.apiextensions.k8s.io "compassconnections.compass.kyma-project.io" already exists
E0122 21:04:26.476985       1 controller.go:287] Helm install error: rpc error: code = Unknown desc = customresourcedefinitions.apiextensions.k8s.io "compassconnections.compass.kyma-project.io" already exist
```

**Steps to reproduce**

1. Install Runtime Agent
2. Delete Helm release
3. Try to install it again

**Comment by janmedrek** [comment]
Owner Reference should be used here.

**Comment by janmedrek** [comment]
Will be solved by #9127
